[FE/07] 로그인 화면

mlnwns opened the pull request:
틀만 잡아둔 상태입니다. 추가하도록 하겠습니다.

loopy-lim committed:
feat: add payment

- add product type
- add payment atom
- add product mock apis
- add react-query call by apis

loopy-lim committed:
Merge pull request #20 from loopy-lim/feature/fe-05

[FE/05] 결제하기 페이지 일부입니다.

loopy-lim committed:
Merge pull request #24 from Step3-kakao-tech-campus/week-04

[Week/04] week 04 develop

loopy-lim committed:
Merge pull request #25 from Step3-kakao-tech-campus/develop

[Develop] week 04 버전입니다.

loopy-lim committed:
Merge branch 'week-04' of https://github.com/Step3-kakao-tech-campus/Team15_FE into feature/fe-05

mlnwns committed:
feat: add login page

mlnwns committed:
feat: add input component

minkonging committed:
feat: 페이머니 내 MyMoney, MoneyHistory 추가

앞으로 출금시 띄워야하는 modal 창 생성 예정
및 페이머니 첫 화면 생성 중

minkonging committed:
feat: 페이머니 페이지에 접속하기 위한 마이페이지 존재 생성

minkonging committed:
feat: 페이머니 초기화면 시작

css 보완 필요

minkonging committed:
feat: 페이머니 출금 modal창 생성 완료

minkonging committed:
feat: 충전 페이지 초기 생성

세부 내역은 수정 예정입니다.

minkonging committed:
feat: 페이머니 페이지에서 충전페이지 이동까지 설정

minkonging committed:
feat: 페이머니 상단부분 완료(잔액조회, 충전, 출금 버튼)

페이머니 상단 부분은 css까지 모두 완료

minkonging committed:
feat: 출금 및 충전 버튼 색 바꿈

mlnwns committed:
feat: add signup page

loopy-lim commented on src/components/common/Input.component.jsx:
```diff
@@ -0,0 +1,6 @@
+export const Input = ({ type, className }) => (
+  <input
+    type={type}
+    className={`border-b py-1 my-4 w-3/4 border-[#62AB05] ${className}`}
+  />
```
input과 label은 친구입니다!
어떻게 쓰면 좋을지 한번 고려해보면 좋을 것 같습니다
또한  type={type}은 너무 많은 것 같습니다... 저희가 쓰는 타입이 무엇인지 확인해보고 분리를 하는 것이 좋을 것 같습니다.

loopy-lim replied:
추가적으로 classnames라는 라이브러리를 사용하면 리터럴로 복잡하게 className을 쓰는 것이 아니라 쉽게 사용할 수 있습니다.

mlnwns replied:
classnames 라이브러리 사용법 숙지하겠습니다!

minkonging replied:
classnames라는 라이브러리는 처음이네요 저도 한번 봐보겠습니다

loopy-lim commented on src/pages/Account.page.jsx:
```diff
@@ -0,0 +1,51 @@
+
+export const AccountPage = () => {
+  return (
+    <>
```
fragment가 왜 밖에 있나요?
노드가 많으면 많을 수록 성능이 느리기 때문에 제거하면 좋을 것 같습니다.

loopy-lim commented on src/pages/Account.page.jsx:
```diff
@@ -0,0 +1,51 @@
+export const AccountPage = () => {
+  return (
+    <>
+      <main
+        aria-label="main"
+        className={classnames(
+          "bg-white overflow-auto",
+          BASE_PADDING,
+          isMobile && isIOS
+            ? "h-[calc(100vh-10rem-env(safe-area-inset-top))]"
+            : "h-[calc(100vh-8.5rem)]"
+        )}
+      >
```
공용 컴포넌트에 MainContainer라는 애가 있습니다!
(너무 같은거 같아 만들어 두었습니다)

mlnwns replied:
공용 컴포넌트가 아직 추가된 상태가 아닌거같아 추후에 추가하도록 하겠습니다

loopy-lim commented on src/pages/Login.page.jsx:
```diff
@@ -0,0 +1,36 @@
+          password
+        </Txt>
+        <Input type="password" />
+        <BottomFullLink title="로그인" to={`/`} />
```
링크가 잘못 되어 있는것 같습니다!

mlnwns replied:
로그인 페이지라 메인화면으로 넘어가도록 했습니다

loopy-lim commented on src/pages/School.page.jsx:
```diff
@@ -0,0 +1,44 @@
+          <Input type="text" />
+        </div>
+
+        <BottomFullLink title="다음으로" to={`/`} />
```
위와 같이 링크가 변경 되어야 할 것 같습니다!

mlnwns replied:
SchoolCard로 링크가 가도록 변경했습니다

loopy-lim reviewed (changes requested):
몇가지가 수정되어야 할 것 같습니다!!
label을 쓰실때 참고로 useId를 이용하면 좋을 것 같습니다.

참고로 reiewers와 assignees, labels를 붙여주시면 감사하겠습니다.
또한 PR를 날리고 있으면 push할 때 동기화가 바로 되어 있어요!(뒤의 PR #29과 겹쳐 말씀드립니다) 
이 PR #27 을 추후 closed하고, 위의 PR로 #29 로 요구사항이 수정된 이후 PR #29 을 merge하겠습니다.

mlnwns committed:
Merge branch 'week-05' into feature/fe-07

minkonging committed:
feat: MyMoney css->tailwind 피드백 수정

loopy-lim committed:
Merge branch 'week-05' of https://github.com/Step3-kakao-tech-campus/Team15_FE into feature/fe-05

minkonging committed:
feat: 페이머니 상단 부분 수정

minkonging committed:
feat: 페이머니 잔액 세부 ui 변경

loopy-lim committed:
feat: add payment page

- change rent calender component (react-calender to date-timepicker)
- add payment anounce component
- add PayButton compoennt

loopy-lim committed:
Merge branch 'week-05' into feature/fe-09

loopy-lim committed:
Merge pull request #28 from minkonging/feature/fe-09

[FE/09] 페이머니 페이지 _ 상단부분 완성

loopy-lim committed:
Merge branch 'week-05' into feature/fe-05

loopy-lim committed:
Merge pull request #30 from loopy-lim/feature/fe-05

[FE/05] 결제 페이지입니다.

mlnwns committed:
feat: add school page

mlnwns committed:
fix: conflict solve

mlnwns committed:
fix: conflict solve
